users_db.py

```diff
@@ -0,0 +1,32 @@
+import sqlite3
+
+
+class Database:
+    def __init__(self, path):
+        self.path = path
+        self.conn = sqlite3.connect(path)
+        self.cursor = self.conn.cursor()
+
+    def create_table(self, table_name):
+        self.cursor.execute(f"""CREATE TABLE if not exists {table_name} 
+                                (username text PRIMARY KEY, password text)""")
+        self.conn.commit()
+
+    def insert_item(self, table_name, payload):
+        self.cursor.execute(f"""INSERT INTO {table_name}(username, password) 
+                                VALUES(?, ?)""", payload)
+        self.conn.commit()
+
+    def get_users_list(self, table_name):
```

Comment:
DB class should not know anything about users and passwords!
Methods should be abstract, it should get all possible info and not be dependent on users and passwords.
User nad passwords is a logic of yyour client-server application, not of the DB class

Reply:
Fixed